features/end_to_end/acas_certificate.feature

```diff
@@ -0,0 +1,40 @@
+@e2e
+@javascript
+Feature:
+  As an ACAS administrator
+  I want to download a certificate/document from the ET Admin
+  So I can validate the certificate number and its contents
+  And view logs detailing user access and error messages
+
+  Scenario: Download and view ACAS Certificate
+    Given I am an ACAS administrator
+    When I enter an ACAS certificate number
```

Comment:
When I enter an ACAS certificate number <-- where? 
What about:
When I enter an ACAS certificate number into the ET Admin

Reply:
When I enter an ACAS certificate number in the ACAS search field?